native/ios/Integreat/Integreat.entitlements

```diff
@@ -4,5 +4,9 @@
 <dict>
 	<key>aps-environment</key>
 	<string>development</string>
+	<key>com.apple.developer.associated-domains</key>
```

Comment:
This will need a change in the profiles of iOS. Did you try to create a production build?

Reply:
No, didn't know about that. What do I have to do?